### ISSUE TITLE
[Lizard]: Organisation Water Authority of Fiji has an ill-formatted unique_id (production)

### ISSUE THREAD

**Issue by GeOdin** [nens/servicedesk-issue-only#133]
### Incident Type

Error

### What priority should we assign to this submission?

None

### Lizard component

Accounts and rights (Default)

### What happened?

I noticed in sentry an error Organisation Water Authority of Fiji has an ill-formatted unique_id.
After asking around. I heared that the software did not check whether the uuid would be correct.
The organisation could be made and does not seem to cause any issues currently, but an ill-formatted unique id might cause problems later (I think),
Would this be something to look into?

Sentry message:
https://sentry.io/organizations/nens/issues/3788455809/?alert_rule_id=260188&alert_timestamp=1670246765401&alert_type=email&environment=production&project=179179&referrer=alert_email

### How to Reproduce?

I am not sure

### Additional info

Good luck! :)

**Comment by GeOdin** [comment]
Note: no prio because it does not seem to cause any issues currently.
But it might be something to look into to prevent ill-formatted uuids to be made.

**Comment by lexvand** [comment]
@GeOdin The UUID is indeed 1 character short. This sometimes happens when copy-pasting it incorrectly. It should be fixed here: https://demo.lizard.net/admin/lizard_auth_client/organisation/597/change/.

If the organisation is also used in 3Di it is important that the UUID's are identical between the two products for postprocessing purposes.

**Comment by GeOdin** [comment]
Ah,thank you very much! :)

**Comment by byrman** [comment]
Linda also bumped into this. It happens when you paste a formatted UUID (i.e. a UUID in its canonical form) in the Django Admin. Because of the field length, the paste action is not completely successful (characters are missing).

Workaround for now:
1. Generate a UUID, e.g. 8e395fa8-9590-4544-a6bf-9ad0fd7e330f
2. Remove all hyphens: 8e395fa895904544a6bf9ad0fd7e330f
3. Paste this hyphen-less value into the Django Admin and save.

**Comment by GeOdin** [comment]
Thank you very much too! :)

Is this something to add to the Lizard docs? Maybe as note?

**Comment by lexvand** [comment]
No, this is not something that normal users or administrators register (yet). It should be checked by the admins that manage organisations.

**Comment by GeOdin** [comment]
Okay, thank you again. :)
